app/uk/gov/hmrc/selfservicetimetopay/controllers/CalculatorController.scala

```diff
@@ -97,11 +97,11 @@ class CalculatorController(calculatorConnector: CalculatorConnector) extends Tim
     sessionCache.get.flatMap {
       case Some(ttpData@TTPSubmission(Some(schedule), _, _, Some(Taxpayer(_, _, Some(sa))), _, _, CalculatorInput(debits, paymentToday, _, _, _, _), _)) =>
         val form = CalculatorForm.createPaymentTodayForm(debits.map(_.amount).sum).fill(paymentToday)
-        Future.successful(Ok(calculate_instalments_form(schedule,
+        Future.successful(Ok(calculate_instalments_form(schedule, Some(sa.debits),
```

Comment:
I want to pass None if the debits are /not/ from the SA, will that case catch it?